.github/workflows/build.yml

```
@@ -78,22 +118,22 @@
         id: cache_graphblas
         uses: actions/cache@v4
         with:
-          path: /FalkorDB/bin/linux-${{ env.ARCH_SUFFIX  }}-release/GraphBLAS
-          key: graphblas-${{ env.ARCH_SUFFIX  }}-${{ hashFiles('./deps/GraphBLAS/Include/GraphBLAS.h') }}
+          path: /FalkorDB/bin/${{ matrix.platform.suffix }}-release/GraphBLAS
+          key: graphblas-${{ matrix.platform.suffix  }}-${{ hashFiles('./deps/GraphBLAS/Include/GraphBLAS.h') }}
 
       - name: Cache parser ${{ matrix.platform }}
```

actionlint (1.7.4)

object, array, and null values should not be evaluated in template with ${{ }} but evaluating the value of type {build_test_image: bool; machine_label: string; name: string; os: string; platform: string; suffix: string; test_dockerfile: string}

(expression)